src/tests/system/tests/test_sss_override.py

```diff
@@ -116,6 +116,61 @@ def test_sss_override__user_attributes(client: Client, provider: GenericProvider
     assert result.home == "/home/o-user1", "User's override name homedir does not match override value!"
 
 
+@pytest.mark.importance("high")
+@pytest.mark.topology([KnownTopology.LDAP, KnownTopology.AD, KnownTopology.Samba])
```

Comment:
`KnownTopology` wasn't backported (yet?) to sssd-2-9* branches...

Reply:
Hi,

thanks for the hint, but I guess the "modern" version should be kept here and backports will need to be fixed.

bye,
Sumit

(Btw, maybe another reason to have a unified test repo?)